src/module.cpp

```diff
@@ -226,71 +226,121 @@ cell AMX_NATIVE_CALL ezhttp_option_set_queue(AMX* amx, cell* params)
 // native EzHttpRequest:ezhttp_get(const url[], const on_complete[], EzHttpOptions:options_id = EzHttpOptions:0);
 cell AMX_NATIVE_CALL ezhttp_get(AMX* amx, cell* params)
 {
+    enum { arg_count, arg_url, arg_callback, arg_option_id, arg_data, arg_data_len };
+
     int url_len;
-    char* url = MF_GetAmxString(amx, params[1], 0, &url_len);
+    char* url = MF_GetAmxString(amx, params[arg_url], 0, &url_len);
 
     int callback_len;
-    char* callback = MF_GetAmxString(amx, params[2], 1, &callback_len);
+    char* callback = MF_GetAmxString(amx, params[arg_callback], 1, &callback_len);
+
+    cell* data = nullptr;
+    const int data_len = params[arg_data_len];
```

Comment:
I don't think need to check arg_count, because `ezhttp_*(const url[], const on_complete[], EzHttpOptions:options_id = EzHttpOptions:0, const data[] = {}, const data_len = 0)`

Reply:
> I don't think need to check arg_count, because `ezhttp_*(const url[], const on_complete[], EzHttpOptions:options_id = EzHttpOptions:0, const data[] = {}, const data_len = 0)`

this is necessary for already compiled plugins, because in them the values for `data` and `data_len` are not defined by default

Reply:
oh i see